python/src/energuide/extracted_datatypes.py

```diff
@@ -506,3 +514,66 @@ def to_dict(self) -> typing.Dict[str, typing.Any]:
             'areaMetres': self.height * self.width,
             'areaFeet': self.height * self.width * _FEET_SQUARED_MULTIPLIER
         }
+
+
+class Ventilation(_Ventilation):
+    _TYPE_MAP = {
+        'false': {
+            'false': ('Heat recovery ventilator', 'Ventilateur-récupérateur de chaleur'),
+            'true': ('Heat recovery ventilator certified by the Home Ventilating Institute',
+                     'Ventilateur-récupérateur de chaleur certifié par le Home Ventilating Institute'),
+        },
+        'true': {
+            'false': ('ENERGY STAR certified heat recovery ventilator',
+                      'Ventilateur-récupérateur de chaleur certifié ENERGY STAR'),
+            'true': ('Home Ventilating Institute listed ENERGY STAR certified heat recovery ventilator',
+                     'Ventilateur-récupérateur de chaleur répertorié par le Home Ventilating Institute et certifié\
+ENERGY STAR'),
+        }
+    }
+
+    @classmethod
+    def _derive_type_string(cls, energy_star: bool, institute_certified: bool) -> typing.Tuple[str, str]:
+        if energy_star and institute_certified:
+            return ('Home Ventilating Institute listed ENERGY STAR certified heat recovery ventilator',
+                    'Ventilateur-récupérateur de chaleur répertorié par le Home Ventilating Institute \
+et certifiéENERGY STAR')
+        elif energy_star and not institute_certified:
+            return ('ENERGY STAR certified heat recovery ventilator',
+                    'Ventilateur-récupérateur de chaleur certifié ENERGY STAR')
+        elif not energy_star and institute_certified:
+            return ('Heat recovery ventilator certified by the Home Ventilating Institute',
+                    'Ventilateur-récupérateur de chaleur certifié par le Home Ventilating Institute')
+        return ('Heat recovery ventilator', 'Ventilateur-récupérateur de chaleur')
+
+
+    @classmethod
+    def from_data(cls, ventilation: element.Element) -> 'Ventilation':
+        energy_star = ventilation.xpath('@isEnergyStar')[0] == 'true'
```

Comment:
We should use `attrib` here. I'll add that to `element.Element`.

Reply:
Does that mean that I should wait for your PR that adds that?